### PR TITLE
routes and controller created

### PR DESCRIPTION
**Pull request by Newprotagonist**
Please review me :)

### CODE REVIEW

**Review by dagmarakorta — APPROVED** [review]
ok, looks good!

File: config/routes.rb [hunk 1]
```diff
@@ -2,7 +2,10 @@
   devise_for :users
   root to: "pages#home"
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :offers do
+    resources :reservations, only: %i[new create]
+  end
+  resources :reservations, only: %i[edit update destroy show index]
```
Comment (timperoulakis) [inline config/routes.rb:8]: no need for index

**Review by timperoulakis — COMMENTED** [review]
no need for reservations#index

**Review by timperoulakis — APPROVED** [review]
Better

**Pull request merged**